crates/json-sql/README.md

```diff
@@ -111,3 +112,45 @@ The list of supported types and corresponding types from [SQL model](../fluvio-m
 | json, jsonb                                 | Json            |
 | uuid                                        | Uuid            |
 
+### Operations
+
+Currently `insert` and `upsert` are supported.
+
+#### Insert
+
+```json
+{
+  "table" : "target_table",
+  "operation": "insert",
+  "map-columns": {
+    "target_column_name" : {
+      "json-key": "device.number",
+      "value": {
+        "type": "int4"
+      }
+    }
+  }
+}
+```
+
+#### Upsert
+
+Upsert additionaly takes an `unique_columns` argument. `unique_columns` specifies the list indices or column names to check for uniqueness of a record.
+If a record with same value in `unique_columns` exists in the database, it will be updated. If no record exists with same value, the given record will
```

Comment:
```suggestion
Upsert additionaly takes an `unique-columns` argument. `unique-columns` specifies the list indices or column names to check for uniqueness of a record.
If a record with same value in `unique-columns` exists in the database, it will be updated. If no record exists with same value, the given record will
```

Reply:
done